lib/fastlane/plugin/test_center/actions/tests_from_xctestrun.rb

```diff
@@ -114,6 +114,12 @@ def self.available_options
             is_string: false,
             default_value: false,
             optional: true
+          ),
+          FastlaneCore::ConfigItem.new(
```

Comment:
You'll also want to add this new option to [`multi_scan`](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/lib/fastlane/plugin/test_center/actions/multi_scan.rb#L346) so that users of it can pass it through

Reply:
Done!